src/smart-components/group/group-table-helpers.js

```diff
@@ -40,44 +44,152 @@ DefaultPlatformPopover.propTypes = {
+                    return {
+                      cells: [
+                        { title: <AppLink to={pathnames['role-detail'].link.replace(':roleId', role.uuid)}>{role.name}</AppLink> },
+                        role.description,
+                        role.accessCount,
+                        <Fragment key={`${uuid}-modified`}>
+                          <DateFormat date={modified} type={getDateFormat(modified)} />
+                        </Fragment>,
+                      ],
+                    };
+                  })}
+                >
+                  <TableHeader />
+                  <TableBody />
+                </Table>
+              ) : (
+                <Text className="pf-u-mx-lg pf-u-my-sm">{intl.formatMessage(messages.noGroupRoles)}</Text>
+              ),
+          },
+        ],
+      },
+      {
+        uuid: `${uuid}-members`,
+        parent: 3 * i,
+        compoundParent: 2,
+        fullwidth: true,
+        noPadding: true,
+        cells: [
+          {
+            props: { colSpan: 7, className: 'pf-m-no-padding' },
+            title:
+              principalCount > 0 ? (
+                <Table
+                  aria-label="Simple Table"
+                  ouiaId="members-in-group-nested-table"
+                  variant={TableVariant.compact}
+                  sortBy={sortByState}
+                  onSort={(index, direction) => {
+                    setSortByState({ index, direction });
+                  }}
+                  cells={[
+                    { title: intl.formatMessage(messages.orgAdministrator) },
+                    { title: intl.formatMessage(messages.firstName), transforms: [sortable] },
+                    { title: intl.formatMessage(messages.lastName), transforms: [sortable] },
+                    { title: intl.formatMessage(messages.username), transforms: [sortable] },
+                    { title: intl.formatMessage(messages.email), transforms: [sortable] },
+                    intl.formatMessage(messages.status),
+                  ]}
+                  rows={members?.map((member) => ({
+                    cells: [
+                      <TextContent key={member.is_org_admin}>
+                        {member?.is_org_admin ? (
+                          <CheckIcon key="yes-icon" className="pf-u-mx-sm" />
+                        ) : (
+                          <CloseIcon key="no-icon" className="pf-u-mx-sm" />
+                        )}
+                        {intl.formatMessage(member?.is_org_admin ? messages.yes : messages.no)}
+                      </TextContent>,
+                      member.first_name,
+                      member.last_name,
+                      member.username,
+                      member.email,
+                      <Label key={member?.is_active} color={member?.is_active && 'green'}>
+                        {intl.formatMessage(member?.is_active ? messages.active : messages.inactive)}
+                      </Label>,
```

Comment:
You could use something like `<ListLoader items={3} isCompact />` for the loading state